.github/workflows/jupyter.yaml

```diff
@@ -1,11 +1,9 @@
 name: "Jupyter Notebook"
 on:
-  push:
+  pull_request:
     paths:
       - 'examples/jupyter'
       - '.github/workflows/jupyter.yaml'
-  release:
-    types: published
```

Comment:
Because it doesn't need to run on release at all. This workflow was very old, and essentially we never looked at it because it only ran on master and for releases, with the latter making not much sense since the workflow did not upload any release artifacts.